File: pset1/cash.c
#include<stdio.h>
#include<math.h>
#include<cs50.h>

int main(void)
{
    float change;
    int coins = 0;
    int change_rem = 0;

    // Prompt user for a positive amount of change
    do
    {
        change = get_float("Change owed: ");
    }
    while (change <= 0.00);

    // Converting dollars to cents
    int cents = round(change * 100);

    // Keeping track of coins and remaining change owed
    coins = cents / 25;
    change_rem = cents % 25;

    coins += change_rem / 10;
    change_rem = change_rem % 10;

    coins = coins + change_rem / 5;
    change_rem = change_rem % 5;

    coins = coins + change_rem / 1;
    change_rem = change_rem % 1;

    printf("%i\n", coins);

}

File: pset6/cash/cash.py
from cs50 import get_float

# Ask user for change
while True:
    change = get_float("Change owed: ")
    if change > 0:
        break

# Converting dollars to cents
cents = round(int(change * 100))

coins = 0

change_rem = cents

# Using lists[]
nominals = [25, 10, 5, 1]
for nom in nominals:
    coins += change_rem // nom
    change_rem %= nom

print(coins)